test/bin.js

```diff
@@ -0,0 +1,11 @@
+import test from 'ava'
+
+test.todo('test bin later')
+
+try {
+  require('./bin') // requiring now for coverage until this is tested
```

Comment:
Could you please update your PR to `require('../bin')` :-) That'll fix the coverage issue :-) Thanks!